frontend/src/pages/upload/Plotting.tsx

```diff
@@ -0,0 +1,68 @@
+// TODO consider using smaller plotly bundle when we will optimize bundle size
+// https://github.com/plotly/plotly.js/blob/master/dist/README.md#plotlyjs-basic
+import Plot from "react-plotly.js";
+import * as Plotly from "plotly.js";
+import React from "react";
+import { CompoundsMap, MeasurementDto } from "../../api/types";
+
+const colors = ["red", "blue", "green"];
+const plotConfig: Partial<Plotly.Config> = {
+  displaylogo: false,
+  // displayModeBar: false,
+};
+
+function hashCode(str: string) {
```

Comment:
What for do you need it? To chose color? If not please add comment. Wy can't you just write `.map(([comp, meas], index) => drawCompoundPlot(comp, meas, index))}` in `UploadPage`?